SUIApp/SUIApp/PurchaseView.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  PurchaseView.swift
+//  SUIApp
+//
+//  Created by Vladka on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора тарифа
+struct PurchaseView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let calendatSysemImageNameText = "calendar"
+        static let bazeTariffText = "Базовый"
+        static let standartTariffText = "Стандарт"
+        static let bisnesTariffText = "Бизнес"
+        static let premiumTariffText = "Премиум"
+        static let tariffText = "Тарифы"
+        static let chooseTariffText = "Выберите подходящий для вас план"
+        static let buyText = "Купить"
+        static let thanksText = "Спасибо за вашу подписку!"
+        static let okText = "Ок"
+        static let packetText = "Пакет:"
+        static let periodTitleText = "Период:"
+        static let priceTitleText = "Стоимость:"
+        static let dollarSymbolText = "$"
+        static let bazePerionText = "7"
+        static let bazePriceText = "10"
+        static let standartPeriodText = "30"
+        static let standartPriceText = "30"
+        static let bisnessPeriodText = "90"
+        static let bisnessPriceText = "100"
+        static let premiumPeriodText = "365"
+        static let premiumPriceText = "300"
+        static let tabSelectionNumber = 2
+        static let cornerRadiusRectangleNumber: CGFloat = 20
+        static let imageSizeNumber: CGFloat = 100
+        static let defaultSpacingNumber: CGFloat = 80
+    }
+    
+    // MARK: - Public Property
```

Comment:
```suggestion
    // MARK: - Body
```
Кидали скрин в важное, как я понял body маскируется отдельно, но могу ошибаться

После боди уже:
// MARK: - Public Property